packages/calypso-e2e/src/lib/blocks/paragraph-block.ts

```diff
@@ -1,7 +1,7 @@
 import { Page, ElementHandle } from 'playwright';
 
 const selectors = {
-	block: '.wp-block-paragraph',
+	block: 'p',
```

Comment:
Previously this did not cause a problem because we never used `ParagraphBlock`'s `validate` method.

Reply:
Oh I meant that I think you could change the selector used in the `validatePublishedContent` to not reference the selector here (since it just adds a `p`) and just look at the text itself alone (without the `p`). But, completely optional 🙂